client/src/js/modules/imaging/views/queuecontainer.js

```diff
@@ -992,12 +999,31 @@ define(['marionette',
             })
 
             this.qsmps.show(this.table2)
+
+            this.plateView = new PlateView({ collection: this.subsamples.fullCollection, type: this.type, inspectionimages: this.inspectionimages })
+            this.listenTo(this.plateView, 'dropClicked', this.filterByLocation, this)
+            this.plate.show(this.plateView)
         },
 
         onShow: function() {
             this.rimg.show(this.image)
         },
         
+        filterByLocation: function(pos) {
+            let newPos = this.avtypeselector.selectedPos != pos
+            this.avtypeselector.selectedPos = null
+            this.avtypeselector._filter()
+            _.map(this.subsamples.fullCollection.where({ isSelected: true }), function(ss) {
```

Comment:
Could we use JS's native `map` instead? Just so we reduce reliance on Underscore:

```suggestion
            this.subsamples.fullCollection.where({ isSelected: true }).map((ss) => {
```